vcpkg.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": [
     "boost-atomic",
     "boost-stl-interfaces",
+    "rsm-binary-io",
     "span-lite",
     "spdlog",
```

Comment:
The only ones we need of these are "boost-stl-interfaces", "rsm-binary-io", and "spdlog".